tasklist/client/src/Tasks/Task/Details/getTaskAssignmentChangeErrorMessage.tsx

```diff
@@ -6,7 +6,13 @@
  * except in compliance with the Camunda License 1.0.
  */
 
-function getTaskAssignmentChangeErrorMessage(code: string) {
+import {t} from 'i18next';
+
+function getTaskAssignmentChangeErrorMessage(
+  code?: string,
+  statusCode?: number,
+  isAssigning?: boolean,
```

Comment:
Why are all these params optional? It's not good to make params optional unless strictly necessary to avoid branching in the code

Reply:
These have been removed after revising the implementation approach.